addons/base_automation/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,40 @@
+---Fields in module 'base_automation'---
+base_automation / base.automation          / _inherits (False)             : NEW 
+base_automation / base.automation          / action_server_id (many2one)   : NEW relation: ir.actions.server, required: required
+base_automation / base.automation          / active (boolean)              : NEW 
+base_automation / base.automation          / filter_domain (char)          : NEW 
+base_automation / base.automation          / filter_pre_domain (char)      : NEW 
+base_automation / base.automation          / last_run (datetime)           : NEW 
+base_automation / base.automation          / on_change_fields (char)       : NEW 
+base_automation / base.automation          / trg_date_calendar_id (many2one): NEW relation: resource.calendar
+base_automation / base.automation          / trg_date_id (many2one)        : NEW relation: ir.model.fields
+base_automation / base.automation          / trg_date_range (integer)      : NEW 
+base_automation / base.automation          / trg_date_range_type (selection): NEW selection_keys: ['day', 'hour', 'minutes', 'month']
+base_automation / base.automation          / trigger (selection)           : NEW required: required, selection_keys: ['on_change', 'on_create', 'on_create_or_write', 'on_time', 'on_unlink', 'on_write']
+base_automation / base.automation.lead.test / active (boolean)              : NEW 
+base_automation / base.automation.lead.test / date_action_last (datetime)   : NEW 
+base_automation / base.automation.lead.test / is_assigned_to_admin (boolean): NEW 
+base_automation / base.automation.lead.test / line_ids (one2many)           : NEW relation: base.automation.line.test
+base_automation / base.automation.lead.test / name (char)                   : NEW required: required
+base_automation / base.automation.lead.test / partner_id (many2one)         : NEW relation: res.partner
+base_automation / base.automation.lead.test / priority (boolean)            : NEW 
+base_automation / base.automation.lead.test / state (selection)             : NEW selection_keys: ['cancel', 'done', 'draft', 'open', 'pending']
+base_automation / base.automation.lead.test / user_id (many2one)            : NEW relation: res.users
+base_automation / base.automation.line.test / lead_id (many2one)            : NEW relation: base.automation.lead.test
+base_automation / base.automation.line.test / name (char)                   : NEW 
+base_automation / base.automation.line.test / user_id (many2one)            : NEW relation: res.users
+base_automation / ir.actions.server        / usage (False)                 : NEW selection_keys: ['base_automation', 'ir_actions_server', 'ir_cron'], mode: modify
```

Comment:
Need to set usage to base_automation for actions that are linked to an action rule

Reply:
Not sure what this usage field is, I'll check that tomorrow

Reply:
In fact we needs to create a ir.actions.server for each base.automation
The field server_action_id being set as delegate=True